gsutil.py

```diff
@@ -16,14 +16,26 @@
 
 """Wrapper module for running gslib.__main__.main() from the command line."""
 
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import division
+from __future__ import unicode_literals
+
 import os
 import sys
 import warnings
 
 # TODO: gsutil-beta: Distribute a pylint rc file.
 
-if not (2, 6) <= sys.version_info[:3] < (3,):
-  sys.exit('gsutil requires python 2.6 or 2.7.')
+if not ((2, 6) <= sys.version_info[:3] < (3,)
+        or (3, 4) <= sys.version_info[:3]):
+    sys.exit('gsutil requires python 2.7 or 3.4+.')
```

Comment:
"3.4+" to me sounds like "3.4 or higher".   Change this to "3.5+"?

Reply:
will do 👍

Reply:
Changed message and logic